bindings/python/pymongocrypt/explicit_encrypter.py

```diff
@@ -17,19 +17,37 @@
 
 
 class ExplicitEncryptOpts(object):
-    def __init__(self, algorithm, key_id=None, key_alt_name=None):
+    def __init__(self, algorithm, key_id=None, key_alt_name=None, index_key_id=None,
+                 query_type=None, contention_factor=None):
         """Options for explicit encryption.
 
         :Parameters:
-          - `algorithm`: The algorithm to use.
+          - `algorithm` (str): The algorithm to use.
           - `key_id`: The data key _id.
           - `key_alt_name` (bytes): Identifies a key vault document by
             'keyAltName'. Must be BSON encoded document in the form:
             { "keyAltName" : (BSON UTF8 value) }
+          - `index_key_id` (bytes): the index key id to use for Queryable Encryption.
+          - `query_type` (int): The query type to execute.
+          - `contention_factor` (int): The contention factor to use
+            when the algorithm is "Indexed".
+
+        .. versionchanged:: 1.3
+           Added the `index_key_id`, `query_type`, and `contention_factor` parameters.
         """
         self.algorithm = algorithm
         self.key_id = key_id
         self.key_alt_name = key_alt_name
+        self.index_key_id = index_key_id
```

Comment:
Should we validate this input as well?

Reply:
Good question. Let's treat it the same way as the existing "key_id" which is validated in PyMongo.